content/tracing/faq/distributed-tracing.md

```diff
@@ -0,0 +1,182 @@
+---
+title: Distributed Tracing
+kind: faq
+further_reading:
+- link: "/tracing/setup/"
+  tag: "Documentation"
+  text: Learn how to setup APM tracing with your application
+- link: "/tracing/visualization/"
+  tag: "Documentation"
+  text: Discover the list of services reporting to Datadog
+- link: "/tracing/visualization/service"
+  tag: "Documentation"
+  text: Learn more about services in Datadog
+- link: "/tracing/visualization/resource"
+  tag: "Documentation"
+  text: Dive into your resource performances and traces
+- link: "/tracing/visualization/trace"
+  tag: "Documentation"
+  text: Understand how to read a Datadog Trace
+---
+
+If you have read the [first example of tracing](/tracing) and want to more deeply understand how tracing works, let's take the following example which represents a simple API **thinker-api** and a micro-service behind it **thinker-microservice**. When the API receives a request with the correct *subject* parameter, it responds with a *thought*, otherwise, it responds with an error:
```

Comment:
`and want to more deeply understand how tracing works,` => `and want understand how tracing works further,`

Reply:
👌